src/dataDisplay/table.jsx

```diff
@@ -1,21 +1,350 @@
+                stickyCellsFirstOfRow[i].style.borderLeft = '';
+                stickyCellsFirstOfRow[i].style.borderRight = STICKY_CELL_BORDER_STYLE;
+            }
+        }
+    }
+
+    parseChildren(children) {
+        const {
+            stickyColumnCount,
+            fixed,
+            singleLine,
+            resizableColumnWidthPercentage,
+        } = this.props;
+        const {
+            adjacentStickyColumnsTotalWidth,
+            stickyTableContainerWidth,
+        } = this.state;
+        const shouldHandleStickyBehavior = !fixed && !singleLine;
+        const parsedChildren = React.Children.map(children, (child) => {
+            if (child.type.name === TABLE_HEADER_TYPE) {
+                const tableHeaderRow = child.props.children;
+                const headerCells = tableHeaderRow.props.children;
+                const parsedHeaderCells = React.Children.map(
+                    headerCells,
+                    (headerCell, index) => {
+                        const isResizable = shouldHandleStickyBehavior &&
+                            (
+                                index === stickyColumnCount - 1 &&
+                                index !== headerCells.length - 1
+                            );
+                        return {
+                            ...headerCell,
+                            props: {
+                                ...headerCell.props,
+                                isResizable,
+                                ...(isResizable && {
+                                    adjacentStickyColumnsTotalWidth,
+                                    stickyColumnCount,
+                                    stickyTableContainerWidth,
+                                    shouldResetWhiteSpaceStyle: shouldHandleStickyBehavior,
+                                    forceTableUpdate: this.forceTableUpdate,
+                                    resizableColumnWidthPercentage,
+                                    // eslint-disable-next-line max-len
+                                    setResizableCellsWordWrapping: this.setResizableCellsWordWrapping,
+                                }),
+                            },
+                        };
+                    },
+                );
+                const parsedTableHeaderRow = {
+                    ...tableHeaderRow,
+                    props: {
+                        ...tableHeaderRow.props,
+                        children: parsedHeaderCells,
+                    },
+                };
+                return {
+                    ...child,
+                    props: {
+                        ...child.props,
+                        children: {
+                            ...parsedTableHeaderRow,
+                        },
+                    },
+                };
+            }
+            return child;
+        });
+        return parsedChildren;
+    }
+
+    applyStickyClassesToCells() {
+        const {
```

Comment:
Looks like we need some security here, because above in the `componentDidMount` it's just making a straight call of this function. No need to step in here if `stickyColumnCount` isn't defined or is 0, right?

Reply:
I already added a validation.